CHANGELOG.md

```diff
@@ -31,7 +31,9 @@ Main (unreleased)
 
 - Flow UI: Add a view for listing the Agent's peers status when clustering is enabled. (@tpaschalis) 
 
-- Add support to the `grafana-agent fmt` CLI for converting a river file from supported formats to river. (@erikbaranowski)
+- Add a new CLI command `grafana-agent convert` for converting a river file from supported formats to river. (@erikbaranowski)
+
+- Add support to the `grafana-agent run` CLI for converting a river file from supported formats to river. (@erikbaranowski)
```

Comment:
duplicated line

Reply:
This includes a wording fix for a previous PR on `grafana-agent convert` that was originally `grafana-agent fmt`.